cleanlab/datalab/datalab.py

```diff
@@ -21,18 +21,25 @@
 """
 from __future__ import annotations
 
+import warnings
```

Comment:
what happens if user specifies: `image_key` but does not have cleanvision dependencies installed?

Reply:
It throws an import error specifying to install image related dependencies or all of them.